src/core/sagas/site.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+
+import { getSiteStatus } from 'core/api/site';
+import { FETCH_SITE_STATUS, loadSiteStatus } from 'core/reducers/site';
+import { getState } from 'core/sagas/utils';
+import type { GetSiteStatusParams } from 'core/api/site';
+import type { Saga } from 'core/types/sagas';
+
+// This saga is not triggered by the UI but on the server side, hence do not
+// have a `errorHandler`. We do not want to miss any error because it would
+// mean no ways for the users to log in, so we let the errors bubble up.
```

Comment:
The concept of "bubble up" is not immediately clear. I suggest explaining it in literal terms: '...it would mean no ways for the users to log in, so we let this saga throw errors without catching them.'